src/app/test-record/defects/defects.component.scss

```diff
@@ -0,0 +1,25 @@
+
+.govuk-summary-list {
+
+  .tag-custom {
+    padding: 1px 10px 1px 10px;
+    vertical-align: middle;
+  }
+
+  & .tag--black {
+    background-color: #0b0c0c !important;
```

Comment:
Is there a way we avoid this? this will make overrides impossible if we have to extend it and also it would probably nice to use a theme variable for the scss background-color, it will make it easier to maintain later.
It is probably worth thinking about a way to scale the scss and architect it.
I think atomic design pattern could be an approach maybe? Thoughts